src/legacy/server/status/routes/api/register_stats.js

```diff
@@ -18,10 +18,12 @@
  */
 
 import Joi from 'joi';
-import { boomify } from 'boom';
+import boom, { boomify } from 'boom';
 import { wrapAuthConfig } from '../../wrap_auth_config';
 import { KIBANA_STATS_TYPE } from '../../constants';
 
+const STATS_NOT_READY_MESSAGE = 'Stats are not ready yet. Please try again later.';
```

Comment:
Can/should we internationalize this string?

Reply:
Yup, thanks for the reminder!